src/panels/config/integrations/dialog-single-config-entry-only.ts

```diff
@@ -0,0 +1,76 @@
+import "@material/mwc-button/mwc-button";
+import { css, CSSResultGroup, html, LitElement, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { HomeAssistant } from "../../../types";
+import { SingleConfigEntryOnlyDialogParams } from "./show-add-integration-dialog";
+
+@customElement("dialog-single-config-entry-only")
```

Comment:
Why did you create a whole new dialog for this? We can just use an alert dialog?

Reply:
No specific reason, just saw how we did it with the YAML dialog. Will change this to the alert dialog later.